cmd/desync/inspectchunks.go

```diff
@@ -0,0 +1,101 @@
+package main
+
+import (
+	"context"
+	"github.com/folbricht/desync"
+	"github.com/spf13/cobra"
+	"io"
+	"os"
+)
+
+type inspectChunksOptions struct {
+	cmdStoreOptions
+	store string
+}
+
+func newinspectChunksCommand(ctx context.Context) *cobra.Command {
+	var opt inspectChunksOptions
+
+	cmd := &cobra.Command{
+		Use:   "inspect-chunks <index> [<output>]",
+		Short: "Inspect chunks from an index and an optional local store",
+		Long: `Prints a detailed JSON with information about chunks stored in an index file.
+By using the '--store' option to provide a local store, the generated JSON will include, if
+available, the chunks compressed size info from that particular store.`,
+		Example: `  desync inspect-chunks file.caibx
+desync inspect-chunks --store /mnt/store file.caibx inspect_result.json`,
+		Args: cobra.RangeArgs(1, 2),
+		RunE: func(cmd *cobra.Command, args []string) error {
+			return runInspectChunks(ctx, opt, args)
+		},
+		SilenceUsage: true,
+	}
+	flags := cmd.Flags()
+	flags.StringVarP(&opt.store, "store", "s", "", "local source store")
+	addStoreOptions(&opt.cmdStoreOptions, flags)
+	return cmd
+}
+
+func runInspectChunks(ctx context.Context, opt inspectChunksOptions, args []string) error {
+	if err := opt.cmdStoreOptions.validate(); err != nil {
+		return err
+	}
+
+	var (
+		outFile io.Writer
+		err     error
+	)
+	if len(args) == 2 {
+		outFileName := args[1]
+		outFile, err = os.Create(outFileName)
+		if err != nil {
+			return err
+		}
+	} else {
+		outFile = stdout
+	}
+
+	// Read the input
+	c, err := readCaibxFile(args[0], opt.cmdStoreOptions)
+	if err != nil {
+		return err
+	}
+
+	var chunksInfo []desync.ChunkAdditionalInfo
+
+	var s desync.LocalStore
+	if opt.store != "" {
+		options, err := cfg.GetStoreOptionsFor(opt.store)
```

Comment:
To improve the UX of this, would you mind initializing the store like in https://github.com/folbricht/desync/blob/master/cmd/desync/prune.go#L50-L58 ?
- Use the existing function to initialize the store
- Then confirm it's a `LocalStore`
- Error if it's something else that's not supported
This should align better with the other commands and provide a clearer error message.